src/test/java/org/folio/marc4ld/mapper/field776/Marc2Bibframe776IT.java

```diff
@@ -82,11 +83,11 @@ void shouldMapField776WithMissingSubfieldsCorrectly() {
       .satisfies(resource -> validateLiteInstance(resource, 5, "work/instance title main title"))
       .satisfies(resource -> validateLocalId(resource, "local ID identifier name"))
       .satisfies(resource -> validateIsbn(resource, "ISBN identifier name"))
-      .satisfies(this::validateUnknown)
-      .satisfies(this::validateLiteInstanceTitle);
+      .satisfies(Marc2Bibframe776IT::validateUnknown)
+      .satisfies(Marc2Bibframe776IT::validateLiteInstanceTitle);
   }
 
-  private void validateLiteWork(Resource resource, String expectedLabel) {
+  public static void validateLiteWork(Resource resource, String expectedLabel) {
```

Comment:
extract these methods to Marc2LdTestBase (or create parent class for 775 and 776 tests extending Marc2LdBase) instead of making them static

Reply:
Done